dpypelines/pipeline/shared/transforms/sdmx/v20.py

```diff
@@ -0,0 +1,71 @@
+# TODO - these functions need creating properly
+# these are just placeholder functions doing very basic things
+# so that they can be picked up by other scripts
+# delete both stub_ functions and change the other functions as required
+# the output paths from smdx_default_v1() were just made up so change as required too
+
+
+import json
+import os
+from pathlib import Path
+
+import pandas as pd
+
+from dpypelines.pipeline.shared.transforms.sdmx.compact.v20.prototype.v1 import (
+    generate_versions_metadata,
+    xmlToCsvSDMX2_0,
+)
+
+
+def sdmx_compact_2_0_prototype_1(input_file: Path):
```

Comment:
assuming im following what you're doing here, you're also going to need an. sdmx_compact_2_1_prototype_1 function.

Reply:
yes, i had created another function `sdmx_generic_2_1_prototype_1` that takes care of the transform for sdmx v2.1